examples/defaulter-gen/Makefile

```diff
@@ -2,7 +2,7 @@ TOOL=defaulter-gen
 
 .PHONY : gen test verify git_verify_clean git_verify_no_change
 test:
-	go test ./_output_tests/marker/...
+	cd _output_tests; go test ./...
```

Comment:
nit but maybe `&&`?

Reply:
Was following the same pattern as the commands for `make gen`. Should both change?

https://github.com/kubernetes/gengo/blob/1d7a6dd710b46db3d7a2f0665f92d7c460c90f74/examples/defaulter-gen/Makefile#L7-L10

Reply:
I would go ahead and change both yeah, thanks.